### PR TITLE
Add MongoDB array of maps values

### PR DESCRIPTION
**Pull request by seeforschauer**
added some data to test $elemMatch and other operators which need embedded structures  for https://github.com/FerretDB/FerretDB/issues/257

### CODE REVIEW

File: cmd/gen/gen.go [hunk 1]
```diff
@@ -65,15 +65,50 @@ func main() {
 		0x0303: {name: "document-two", v: map[string]any{"document": 42.13, "foo": "bar"}},
 		0x0304: {name: "document-three", v: map[string]any{"document": int32(0), "baz": nil}},
 
+		// for projection tests
```
Comment (AlekSi) [inline cmd/gen/gen.go]: test_db is a separate project used by others. That comment would be very confusing for them :)

I think the values dataset should be as diverse as possible, but also as small as possible while being useful

Comment (seeforschauer) [reply cmd/gen/gen.go]: removed.
also removed documents that aren't in use for tests now.

File: cmd/gen/gen.go [hunk 2]
```diff
@@ -65,15 +65,20 @@ func main() {
 		0x0303: {name: "document-two", v: map[string]any{"document": 42.13, "foo": "bar"}},
 		0x0304: {name: "document-three", v: map[string]any{"document": int32(0), "baz": nil}},
 
+		0x0305: {name: "document-four", v: map[string]any{"code": 121081}},
+		0x0306: {name: "document-five", v: []any{
```
Comment (AlekSi) [inline cmd/gen/gen.go]: Value here is an array, not a document, so it should be "array-XXX", not "document-XXX"

Comment (seeforschauer) [reply cmd/gen/gen.go]: changed

File: cmd/gen/gen.go [hunk 3]
```diff
@@ -69,10 +69,9 @@ func main() {
 		0x0402: {name: "array-empty", v: []any{}},
 		0x0403: {name: "array-one", v: []any{42.13}},
 		0x0404: {name: "array-three", v: []any{42, "foo", nil}},
-		0x0405: {name: "array-embedded", v: []any{
```
Comment (AlekSi) [inline cmd/gen/gen.go]: Ok, but maybe we could use that existing document? Why change it? I think it is already used somewhere

Comment (seeforschauer) [reply cmd/gen/gen.go]: I checked it's not used in dance and in FerretDB. 
If you know any other places, then it to be left here w/o changes. Do you? 
And add new other ID.

Comment (seeforschauer) [reply cmd/gen/gen.go]: reverted

**Comment by AlekSi** [comment]
Not merging this one as FerretDB moves away from using test_db, and test_db itself does not need that data

**Pull request closed without merging**